ci/Dockerfile

```diff
@@ -14,6 +14,20 @@ RUN apt-get update && apt-get install -y wget jq git vim nano python3-pip\
 COPY scripts/requirements.txt /requirements.txt
 RUN /usr/bin/python3 -m pip install -r /requirements.txt
 
+RUN echo "# Use local bosh release -- useful for testing \
```

Comment:
why not make this a local ops file and copy it it to `/usr/local/local-releases.yml` as a COPY step instead of this inline RUN step? would make the Dockerfile less clustered

Reply:
Yes that's fair. Will do. This file at some point disappeared form the `main-bosh-docker` parent image. I added this as quick workaround to keep going and it stayed like this.

Reply:
You are calling `sponge` command which is not available by default. We should install it before running the script.

Reply:
> You are calling `sponge` command which is not available by default. We should install it before running the script. 

We might not even need it. I'll test with a simple redirect and add a change.

Reply:
I've removed the file altogether as it wasn't really used. The file that was written in the Dockerfile was actually not consistent (i.e. it referenced a variable that is never set) but was seemingly ignored.
The newer versions of our base image `bosh/main-bosh-docker` don't have the local-releases.yml opsfile anymore. I've removed it from our copy of the `start-bosh` script as well now. We should consider reverting back to the `start-bosh` script of the underlying image again at some point.